CHANGELOG.md

```diff
@@ -20,6 +22,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 - Allow one simple text plugin as person page main content
 - Allow glimpse plugins in course page information placeholder
 - Revert to long plugin names in side toolbar placeholder plugin list
+- Update translation files
```

Comment:
we normally don't put it in the change log as it is required to be done before each release.

Reply:
Ok i'll remove it then